src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -1655,7 +1655,8 @@ void ElectricEIRChillerSpecs::size(EnergyPlusData &state)
 
     if (state.dataPlnt->PlantFinalSizesOkayToReport) {
         if (this->IPLVFlag) {
-            Real64 IPLV = 0.0;
+            Real64 IPLVSI = 0.0;
+            Real64 IPLVIP = 0.0;
```

Comment:
Now chiller's IPLV calculation function returns IPLVs determined using the test conditions specified in 2011 AHRI Std 550/590 for IP and 2011 AHRI Std 551/591 for SI.